index.html

```diff
@@ -74,15 +74,14 @@
 ```
 2. Create bandwidth limit rule:
 ```bash
-neutron qos-bandwidth-limit-rule-create \
-    --max-kbps 512 \
-    --max-burst 1024 \
-    example-policy
+neutron qos-bandwidth-limit-rule-create --max-kbps 512 \
+                                            --max-burst 1024 \
```

Comment:
without this extra indentation in source, it was showing under indented in browser. I hope it's ok.